index.html

```diff
@@ -172,7 +177,7 @@ <h3>Multi-Post
         </div>
       </div>
     </section>
-    <section class="about" id="about">
+  <section class="about" id="about">
```

Comment:
The About section does not follow the design on laptop view. Kindly see the screenshot below.
![about](https://user-images.githubusercontent.com/35504149/135631229-8975c718-be77-462c-a5a7-72faa7a1cef1.png)